lib/src/main/java/com/thenewboston/data/dto/bankapi/accountdto/AccountDTO.kt

```diff
@@ -0,0 +1,15 @@
+package com.thenewboston.data.dto.bankapi.accountdto
+
+import com.google.gson.annotations.SerializedName
+import java.util.*
+
+data class AccountDTO(
+    val id: String,
+    @SerializedName("created_date")
+    val createdDate: Date,
```

Comment:
should we use Date ? maybe something more modern

Reply:
@akulinski  I agree, let's tackle the Dates with a separate Issue